config/queue-monitor.php

```diff
@@ -20,7 +20,7 @@
     // The optional UI settings.
     'ui' => [
         // Enable the UI
-        'enabled' => false,
+        'enabled' => env('QUEUE_MONITOR_UI_ENABLED', false),
```

Comment:
I would want to introduce a new env var at this point, maybe in a future major release

```suggestion
        'enabled' => false,
```